apps/src/code-studio/levels/multi.js

```diff
@@ -286,6 +314,12 @@ Multi.prototype.submitButtonClick = function () {
     return;
   }
 
+  if (!this.allowMultipleAttempts) {
+    this.lockAnswers();
+    $('.submitButton')?.hide();
+    $('.nextLevelButton')?.show();
```

Comment:
my first time seeing this `?.` syntax with a jquery selector, and it makes so much sense!